src/Monitors/Models2.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Collection\Monitors;
```

Comment:
You should remove this since it's not being used

Reply:
Not sure what you mean here as I am not using the file, should I have ignored the file